jenkins_jobs/upload_iso/script.sh

```diff
@@ -13,5 +13,4 @@ rsync_upload() {
 mkdir iso
 rsync_upload --recursive iso
 
-rsync_upload *.iso
-rsync_upload *-CHECKSUM
+rsync_upload result/iso/latest/*
```

Comment:
The archive above (`rsync_upload result/iso/latest/*`) is not copying the directory structure, so you do not have to change this.